3-mcstas/QENS_from_function.ipynb

```diff
@@ -757,57 +760,60 @@
     "- `sample_distance`: 150 m\n",
     "- `energy_width_ueV`: 5 ueV\n",
     "- `sample_choice`: '\"Elastic\"'\n",
-    "- `frequency_multiplier`: 10 (This controls the ratio between chopper and source frequency)"
+    "- `frequency_multiplier`: 10 (This controls the ratio between chopper and source frequency)\n",
+    "\n",
+    "Use the `settings` method to set a reasonable name, including the sample type and number of pulses currently used.\n",
+    "\n",
+    "Store the resulting data in a variable called `data_improved`."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "complete-norwegian",
+   "id": "reasonable-story",
    "metadata": {
     "slideshow": {
      "slide_type": ""
     },
-    "tags": []
+    "tags": [
+     "solution",
+     "hide-cell"
+    ]
    },
    "outputs": [],
    "source": [
-    "instrument.show_parameters()"
+    "instrument.settings(output_path=\"QENS_elastic_1_pulse\")\n",
+    "instrument.set_parameters(energy_width_ueV=5, sample_choice='\"Elastic\"', n_pulses=1, frequency_multiplier=10)\n",
+    "\n",
+    "data_improved = instrument.backengine()"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "reasonable-story",
+   "cell_type": "markdown",
+   "id": "multiple-balance",
    "metadata": {
-    "slideshow": {
-     "slide_type": ""
-    },
-    "tags": []
+    "tags": [
+     "remove-cell",
+     "dmsc-school-keep"
+    ]
    },
-   "outputs": [],
    "source": [
-    "instrument.settings(\n",
-    "    output_path=\"QENS_elastic_1_pulse\",\n",
-    ")\n",
-    "instrument.set_parameters(\n",
-    "    energy_width_ueV=5,\n",
-    "    sample_choice='\"Elastic\"',\n",
-    "    n_pulses=1,\n",
-    "    frequency_multiplier=10,\n",
-    ")\n",
-    "\n",
-    "data_improved = instrument.backengine()"
+    "We will be reading values of the produced graphs, in those cases the %matplotlib widget mode helps as it provides information on the cursor position over the graphs."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "liberal-venezuela",
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "remove-cell",
+     "dmsc-school-keep"
+    ]
+   },
    "outputs": [],
    "source": [
-    "# %matplotlib widget"
+    "%matplotlib widget"
```

Comment:
If you enable the `widget` backend, then the figures won't show up in the jupyter book online, that's why I commented it out, so that the students can easily uncomment it to get interactive figures.